app/src/filters/components/FilterGroupItem.jsx

```diff
@@ -48,16 +51,23 @@ class FilterGroupItem extends Component {
       </ul >
     );
 
+    const tooltip = (this.props.filterGroup.label.length > 30) ? this.props.filterGroup.label : null;
```

Comment:
I’m wondering if we should make a reusable component out of this “if-short-then-label-else-tooltip” thing.  May be useful for pinned vessel names and such. No need to do it now, just a suggestion. Thoughts?

Reply:
Yes probably something like a `<Tooltipable />` higher-order component or the like. Let's keep that in mind.